apps/explorer/lib/explorer/application.ex

```diff
@@ -152,6 +152,7 @@ defmodule Explorer.Application do
         Explorer.Migrator.BackfillMultichainSearchDB
         |> configure_mode_dependent_process(:indexer)
         |> configure_multichain_search_microservice(),
+        configure(Explorer.Migrator.ArbitrumDaRecordsNormalization),
```

Comment:
Should we run this migration only on the indexer pod?
```suggestion
        configure(Explorer.Migrator.ArbitrumDaRecordsNormalization, :indexer),
```

Reply:
Done in f50c8bd